relay-v3/README.md

```diff
@@ -0,0 +1,55 @@
+# Requirements
+
+- **CNI** installed
+- **Docker** installed
+- **Relay-v3** container image
+- **OpenSVC** lastest version
+
+# Deploy the relay secrets
+
+```
+om system/sec/relay-v3 create
+om system/sec/relay-v3 add --key users/myusername --value mypassword
+```
+
+# Deploy the relay service
+
+**Note**: You can find how to build the relay-v3 container image here: https://github.com/opensvc/docker_osvc_relay-v3
+
+```
+om system/svc/relay-v3 create
+om system/svc/relay-v3 deploy --config https://raw.githubusercontent.com/opensvc/opensvc_templates/main/relay-v3/relay-v3.conf
+```
+
```

Comment:
Add a relay-v3-ip-cni.conf version for this first deploy command.
Use relay-v3.conf for the custom network config option.

Reply:
[ip#0]
netns = container#0
type = cni
network = default
wait_dns = 1m